src/common/rivetsExtra.es6

```diff
@@ -60,18 +60,20 @@ rv.formatters['and-not'] = (vlaue, other) => vlaue && !other;
 rv.formatters['gt'] = (vlaue, other) => vlaue > other;
 /* rivets formatter for < operator  */
 rv.formatters['lt'] = (vlaue, other) => vlaue < other;
+/* rivets formatter to check if a variable has value */
+rv.formatters['has-value'] = (value) => value === 0 ? true : Boolean(value);
 /* localise price format*/
-rv.formatters['format-price'] = (value, currency) => (value) ?  formatPrice(value, currency) : undefined;
+rv.formatters['format-price'] = (value, currency) => (!isNaN(value)) ?  formatPrice(value, currency) : undefined;
```

Comment:
Can we use  Number.isNaN() and I think we still need to check `(value) ?  formatPrice(value, currency) : ...`?

Reply:
Okay, changed it to a new function